h2o-persist-s3/src/main/java/water/persist/PersistS3.java

```diff
@@ -259,8 +255,8 @@ public void importFiles(String path, String pattern, ArrayList<String> files, Ar
     }
   }
 
-  private static void ignoreAndWait(final Exception e, boolean printException) {
-    Log.ignore(e, "Hit the S3 reset problem, waiting and retrying...", printException);
+  private static void ignoreAndWait(final Exception e) {
+    LOGGER.debug("Hit the S3 reset problem, waiting and retrying...", e);
```

Comment:
I always want to print it. Former ignore was `debug` anyway. Why not print the exception all the time in debug ?